test/callback.js

```diff
@@ -22,13 +22,6 @@ describe('weak()', function () {
       assert(called)
     })
 
-    it('should pass the target in as the first argument', function () {
-      weak({ foo: 'bar' }, function (o) {
```

Comment:
The persistent gets deallocated before the weak callback executes, it is not allowed to access it.